phpunit/blocks/render-comment-template-test.php

```diff
@@ -142,6 +142,9 @@ public function test_inner_block_inserted_by_render_block_data_is_retained() {
 			return $parsed_block;
 		};
 
+		// Remove auto-insertion filter so it won't collide.
+		remove_filter( 'render_block_data', 'gutenberg_auto_insert_child_block' );
```

Comment:
Looks like the changes to this file are from an earlier experiment? (`gutenberg_auto_insert_child_block` doesn't appear to be present in this change)

Reply:
Indeed! Good spot, thank you very much! 

Fixed in 334dd9939c.